notebooks/406-human-pose-estimation-3d/406-opencv-human-pose-estimation-3d.ipynb

```diff
@@ -0,0 +1,597 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4031f1e0-69b4-467b-b09d-1eac80fd6609",
+   "metadata": {},
+   "source": [
+    "# Live 3D Human Pose Estimation with OpenVINO\n",
+    "\n",
+    "This notebook demonstrates a live 3D Human Pose Estimation webcam example with OpenVINO. We utilize the model [human-pose-estimation-3d-0001](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/human-pose-estimation-3d-0001) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). At the last session of this notebook, you will see the live inference results from your webcam (if available). Alternatively, you can also upload a video file to test out the algorithms.\n",
```

Comment:
Please change the description here accordingly. Also, please point out what's the difference between this notebook and the other notebook demo in the same repo, to make it more clearly to the developers.